commands/maliciousscan/maliciousscan.go

```diff
@@ -0,0 +1,282 @@
+		}()
+		for _, targetResult := range cmdResults.Targets {
+			if targetResult.AppsConfigModule == nil {
+				_ = targetResult.AddTargetError(fmt.Errorf("can't find module for path %s", targetResult.Target), false)
+				continue
+			}
+			appsConfigModule := *targetResult.AppsConfigModule
+			jasParams := runner.JasRunnerParams{
+				Runner:         &jasScanProducerConsumer,
+				ServerDetails:  serverDetails,
+				Scanner:        scanner,
+				Module:         appsConfigModule,
+				ScansToPerform: []utils.SubScanType{utils.MaliciousCodeScan},
+				ScanResults:    targetResult,
+				TargetCount:    len(cmdResults.Targets),
+			}
+
+			if generalError = runner.AddJasScannersTasks(jasParams); generalError != nil {
+				_ = targetResult.AddTargetError(fmt.Errorf("failed to add malicious scan task: %w", generalError), false)
+				generalError = nil
+			}
+		}
+		return
+	}
+
+	if _, addTaskErr := jasScanProducerConsumer.Runner.AddTaskWithError(createMaliciousScansTask, func(taskErr error) {
+		cmdResults.AddGeneralError(fmt.Errorf("failed while adding JAS scan tasks: %s", taskErr.Error()), false)
+	}); addTaskErr != nil {
+		return fmt.Errorf("failed to create JAS task: %w", addTaskErr)
+	}
+
+	jasScanProducerConsumer.Start()
+
+	if cmd.progress != nil {
+		if err = cmd.progress.Quit(); err != nil {
+			return err
+		}
+	}
+
+	if err = output.NewResultsWriter(cmdResults).
+		SetOutputFormat(cmd.outputFormat).
+		SetPlatformUrl(cmd.serverDetails.Url).
+		SetPrintExtendedTable(false).
+		SetIsMultipleRootProject(cmdResults.HasMultipleTargets()).
+		SetSubScansPerformed([]utils.SubScanType{utils.MaliciousCodeScan}).
+		PrintScanResults(); err != nil {
+		return errors.Join(err, cmdResults.GetErrors())
+	}
+
+	if err = cmdResults.GetErrors(); err != nil {
+		return err
+	}
+
+	log.Info("Malicious scan completed successfully.")
+	return nil
+}
+
+func logScanPaths(workingDirs []string) {
+	if len(workingDirs) == 0 {
+		return
+	}
+	if len(workingDirs) == 1 {
+		log.Info("Scanning path:", workingDirs[0])
+		return
+	}
+	log.Info("Scanning paths:", strings.Join(workingDirs, ", "))
+}
+
+func populateScanTargets(cmdResults *results.SecurityCommandResults, workingDirs []string) {
+	for _, requestedDirectory := range workingDirs {
+		if !fileutils.IsPathExists(requestedDirectory, false) {
+			log.Warn("The working directory", requestedDirectory, "doesn't exist. Skipping scan...")
+			continue
+		}
+		cmdResults.NewScanResults(results.ScanTarget{Target: requestedDirectory, Name: filepath.Base(requestedDirectory)})
+	}
+
+	if len(workingDirs) == 0 {
+		currentDir, err := coreutils.GetWorkingDirectory()
+		if err != nil {
+			cmdResults.AddGeneralError(fmt.Errorf("failed to get current working directory: %w", err), false)
+			return
+		}
+		cmdResults.NewScanResults(results.ScanTarget{Target: currentDir, Name: filepath.Base(currentDir)})
+	}
+
+	if len(cmdResults.Targets) == 0 {
+		log.Warn("No scan targets were detected. Proceeding with empty scan...")
```

Comment:
```suggestion
		log.Warn("No scan targets were detected.")
```
you are returning here....